source/community/release_notes/ulmo/ulmo_catalog.rst

```diff
@@ -0,0 +1,26 @@
+.. _Ulmo catalog:
+
+Course Catalog Revamp
+######################
+
+In the Ulmo release, a new and improved Course Catalog is available. This new
+catalog experience features an updated UI, better searching and sorting, and
+highly customizable Catalog and Course About pages.
+
+Visual Improvements
+********************
+
+(insert screenshots)
+
+Customizability
+*****************
+
+(link to available plugin slots)
+
+Ready To Try It Out?
+**********************
+
+Site operators should see *these (link)* instructions to install the new Course
```

Comment:
This link will be the README once https://github.com/openedx/frontend-app-catalog/pull/60 is merged.